app/journey/CouncilTaxBandType.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package journey
+
+import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue}
+
+sealed trait CouncilTaxBandType
+
+case object A extends CouncilTaxBandType
```

Comment:
I don't know what is better. Java `Enum` or scala sealed trait and case object?

Reply:
I think the scala sealed trait one ! But also that's what in used for the other enum, so it s better to use sealed traits for consistency!